packages/web/src/models/pool/mapper/pool-mapper.ts

```diff
@@ -95,15 +95,15 @@ export class PoolMapper {
       tokenA,
       tokenB,
       feeTier: feeTierInfo?.type || "NONE",
-      apr: apr !== null ? apr.toString() : "",
+      apr: (apr ?? "").toString(),
```

Comment:
The modification of the APR should be done in the `fromResponse` `detailFromResponse` functions, not in this part.
You can use `totalApr` instead of `apr`. 

```
public static fromResponse(pool: PoolResponse): PoolModel {
    const id = pool.id ?? makeId(pool.poolPath);
    return {
      ...pool,
      id,
      incentiveType: pool.incentiveType as INCENTIVE_TYPE,
      rewardTokens: pool.rewardTokens || [],
      apr: pool.totalApr,
      bins: pool?.bins || [],
      bins40: pool?.bins40 || [],
      liquidity: pool.liquidity,
      allTimeVolumeUsd: pool.allTimeVolumeUsd,
      price: Number(pool.price),
    };
  }
```

Reply:
I will working this together with api migration